bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -148,9 +148,9 @@ BCStateTran::BCStateTran(const Config &config, IAppState *const stateApi, DataSt
       numberOfReservedPages_{0},
       randomGen_{randomDevice_()},
       sourceSelector_{SourceSelector(
-          allOtherReplicas(), config_.fetchRetransmissionTimeoutMilli, config_.sourceReplicaReplacementTimeoutMilli)},
+          allOtherReplicas(), config_.fetchRetransmissionTimeoutMs, config_.sourceReplicaReplacementTimeoutMs)},
```

Comment:
Nit: `SourceSelector` in `sourceSelector_{SourceSelector(` seems redundant.

Reply:
done